src/store/sagas/ethereum/transactions.ts

```diff
@@ -1,15 +1,16 @@
 import { put, takeEvery } from 'redux-saga/effects';
 
+import { doGet } from '../../helpers/fetch';
 import { FetchTransactionsAction } from '../../actions/types';
 import { Transactions } from '../../../types';
 import * as ethereumActions from '../../actions/ethereum/transactions';
 
 async function fetchTransactions(walletHash: string, page: number, resultsPerPage: number = 20): Promise<Transactions> {
-  const res = await fetch(
-    /* eslint-disable-next-line max-len */
-    `${process.env.REACT_APP_HOST}/api/v1/ethereum/transactions?groupBy=buyer&resultsPerPage=${resultsPerPage}&page=${page}&walletHash=${walletHash}`
-  );
-  return res.json();
+  const url = `api/v1/ethereum/transactions?groupBy=buyer&resultsPerPage=${resultsPerPage}&page=${page}&walletHash=${walletHash}`;
```

Comment:
Nad tą linią trzeba dodać 
   /* eslint-disable-next-line max-len */
co by eslint nie krzyczał, a po co znowu robić warningi

Reply:
hmm 🤔 to po co mamy tą regułę jeśli potrzebujemy więcej miejsca

Reply:
Można podać inną długość, nawet 200 znaków, jeśli to nam ułatwi życie :) Ale w praktyce coś co ma więcej niż 120-150 trzeba złamać. Chyba, że się nie da, to się nie da, ale to rzadkie przypadki (takie jak ten ^^).